frontend/src/features/directory/components/FilterableVendorTable.tsx

```diff
@@ -33,7 +33,7 @@ function FilterableVendorTableContent({ uniqueRegions, tags, vendors, favoriteVe
   const selectedRegion = searchParams.get(LOCATION_PARAM) || "";
   const searchQuery = searchParams.get(SEARCH_PARAM) || "";
   const travelsWorldwide = searchParams.get(TRAVEL_PARAM) === "true";
-  const selectedSkill = searchParams.get(SKILL_PARAM) || "";
+  const selectedSkills = useMemo(() => searchParams.getAll(SKILL_PARAM) || [], [searchParams]);
```

Comment:
should all the searchParam-derived constants have useMemo()?

Reply:
this was suggested by the IDE but I think only skills needs to be bc it's a list!